### PR TITLE
Cnft1 1162 patient profile geocoding and address verification UI

### PR DESCRIPTION
**Pull request by GangulaMounika**

### CODE REVIEW

File: apps/modernization-ui/src/pages/patient/profile/addresses/AddressEntryForm.tsx [hunk 1]
```diff
@@ -49,256 +162,406 @@ export const AddressEntryForm = ({ action, entry, onChange, onCancel }: EntryPro
...
+                                                <li key={idx} className="usa-nav__submenu-item">
+                                                    <SearchCriteriaContext.Consumer>
+                                                        {({ searchCriteria }) => {
+                                                            return (
+                                                                <Button
+                                                                    key={idx}
+                                                                    onClick={() =>
+                                                                        handleSuggestionSelection(
+                                                                            idx,
+                                                                            searchCriteria.states
```
Comment (adamloup-enquizit) [inline apps/modernization-ui/src/pages/patient/profile/addresses/AddressEntryForm.tsx:271]: The state list is available in the component in `location.states`.  Trying to keep `SearchCriteriaContext` use limited in favor of the hooks that pull multiple lists at once.

**Review by adamloup-enquizit — CHANGES_REQUESTED** [review]
`const location = useLocationCodedValues();` this hook loads the `states` and `countries` lists and should be used.  The Patient Profile isn't wrapped in a `SearchCriteriaContext` so any lists pulled from there will be empty.

File: apps/modernization-ui/src/pages/patient/profile/addresses/AddressEntryForm.tsx (hunk 1 above)
Comment (GangulaMounika) [reply apps/modernization-ui/src/pages/patient/profile/addresses/AddressEntryForm.tsx]: Sounds good.

**Review by adamloup-enquizit — APPROVED** [review]
Looks good! 👍

**Pull request merged**